### DISCUSSION TITLE
report.gg_matrix.tsv vs report.tsv

### DISCUSSION THREAD

**Discussion by AnirudhKashyap511** [vdemichev/DiaNN#1230] [Q&A]
What does it mean if the report.tsv file has data but report.gg_matrix.tsv has no data, could the timsTOF file be corrupted, which means that I need to run the data and diann again or does it mean somethign else. Also, wanted to know how changing the DIANN versions would change protein abundance if it changes at all?

Thank you so much!

**Comment by vdemichev** [comment]
Hi,

Possible reasons: no FASTA provided, no genes passed q-value filtering. How does the log look like?

> could the timsTOF file be corrupted

No.

> how changing the DIANN versions would change protein abundance

Protein abundance is not comparable between versions.

Best,
Vadim

**Reply by AnirudhKashyap511** [reply]
[9:14] Number of genes identified at 1% FDR: 263 (precursor-level), 0 (protein-level) (inference performed using proteotypic peptides only)
Figured why the report.gg_matrix is empty. Was wondering in which scenario does this happen?

**Reply by vdemichev** [reply]
> no genes passed q-value filtering

Likely this

**Reply by AnirudhKashyap511** [reply]
Thank you for your prompt reply, I had another question if you dont mind. I was trying to understand my data and PG.Quantity	PG.Normalised	PG.MaxLFQ	Genes.Quantity	Genes.Normalised	Genes.MaxLFQ	Genes.MaxLFQ.Unique
 all showed similar values so was wondering what that could mean?

**Reply by vdemichev** [reply]
> all showed similar values

That's fine. Can even be identical if there's a single run & single precursor matching the protein.

**Reply by AnirudhKashyap511** [reply]
Thanks! Makes sense, just had one last question. How is protein abundance calculated? Is Ms1Area being summed? I tried it it doesnt match.

**Reply by vdemichev** [reply]
Based on Precursor.Normalised, using MaxLFQ (Legacy) or QuantUMS (QuantUMS mode)
